CHANGELOG.md

```diff
@@ -1,11 +1,39 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+### Added
+- Alignment position support (#44)
```

Comment:
Since the whole point of this hotfix is to revert this change, it shouldn't be in the changelog.

Reply:
Good catch.

Reply:
This was resolved by code review.